HACKING.md

```diff
@@ -1,21 +1,35 @@
-Mir hacking guide
-=================
+# Mir hacking guide
 
-Coding *Mir*
-----------
+## Building & running
 
-There's a *__coding style guide__* in the guides subdirectory. To build it into an
+There are some brief guides describing how to run the *Mir binaries* once you have
+them built. You might think it's obvious but there are some important things
+you need to know to get it working, and also to prevent your existing *X server*
+from dying at the same time.
```

Comment:
Why discuss running the binaries you have built before discussing building them?

Reply:
Agreed. I felt like this belonged at the tom of the `HACKING.md` file (i.e. how to build and run), but perhaps the wording was a little weird.